rust/src/router/plain_transport.rs

```diff
@@ -380,13 +380,13 @@ impl Inner {
             if close_request {
                 let channel = self.channel.clone();
                 let router_id = self.router.id();
-                let request = TransportCloseRequest {
+                let request = TransportCloseRequestNew {
```

Comment:
`Fbs` would be a more appropriate suffix, it was already used in various places.